app/views/protocols/index/_protocol_versions_modal.html.erb

```diff
@@ -8,7 +8,8 @@
         </h4>
       </div>
       <div class="modal-body">
-        <% if @protocol.draft.present? %>
+        <% draft_present = @protocol.draft.present? %>
```

Comment:
This logic should probably be implemented inside the appropriate permission helper.